<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>performans_rapor</class>
 <widget class="QMainWindow" name="performans_rapor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>410</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>410</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>410</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Rapor EkranÄ±</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>580</width>
      <height>341</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>360</y>
      <width>131</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 173, 181);</string>
    </property>
    <property name="text">
     <string>OK</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="enabled">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>performans_rapor</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>717</x>
     <y>487</y>
    </hint>
    <hint type="destinationlabel">
     <x>796</x>
     <y>361</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
